byterun/array.c

```diff
@@ -12,7 +12,7 @@
 /***********************************************************************/
 
 /* Operations on arrays */
-
+#include <stdio.h>
```

Comment:
It is not, I forgot to remove it after using printf for debugging purpose (I tried to see if I was able to raise out_of_memory between each call). Sorry :-/.